airflow/providers/sendgrid/PROVIDER_CHANGES_0.0.1.md

```diff
@@ -0,0 +1,12 @@
+
+
+### Release 1.0.0b2
```

Comment:
This file and the version number don't match  -- does that matter?

Reply:
Yep. I have to add proper versioning support (that's one of the remaining tasks)